### PR TITLE
Adobe Flash Cheat sheet : Alias added

### PR DESCRIPTION
**Pull request by ankushg07**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [ ] New Instant Answer
    - [ ] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [x] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [x] Enhancement: **`{Adobe Flash Cheat Sheet}: {Added alias for the query }`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

What changes does this PR introduce? 
This will trigger the cheat sheet using the alias or common names 
###### Related Issues or Discussions 

---

Instant Answer Page :https://duck.co/ia/view/adobe_flash_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@ankushg07 Thanks for the pull request! Unfortunately, we're only accepting contributions and improvements to a limited portion of the search space.
            
The staff and community are currently focused on making DuckDuckGo the [best search engine for programmers](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53). To ensure we're making as much progress as possible, **we are only accepting Instant Answers and updates related to the programming mission**.
    
If this is not related to the Programming Mission, it will likely be closed but you can:
    
1. Help us prevent further confusion by telling us where you found out about DuckDuckHack or where we could've alerted you earlier.
2. Keep this code handy! We'll be expanding into other search topics in the future!
3. Join us at our new home: https://forum.duckduckhack.com/

If this is related to the Programming Mission, it will be reviewed by staff and the community as soon as possible!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/adobe-flash.json [hunk 1]
```diff
@@ -6,6 +6,11 @@
       "sourceName":"Academia",
       "sourceUrl":"http://www.academia.edu/13246006/Adobe_Flash_CC_2015_-keyboard-shortcuts-cheat-sheet"
    },
+   "aliases": [
+	    "adobe flash cs6",
+		"flash cs6",
```
Comment (sahildua2305) [inline share/goodie/cheat_sheets/json/adobe-flash.json]: nit: indentation issues

Comment (ankushg07) [inline share/goodie/cheat_sheets/json/adobe-flash.json]: @sahildua2305 Well it looks fine in editor 
Still i will re-align it and commit

**Comment by ankushg07** [comment]
@sahildua2305  looks fine to me now

**Comment by gautamkrishnar** [comment]
@ankushg07 thanks for taking your time to contribute 👍

**Pull request merged**